.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,7 @@ Summary of changes:
 I made sure to:
 - [ ] Add and / or update tests
 - [ ] Run the code formatter (`CTRL + ALT + L`)
-- [ ] Update any documentation (javadoc, domain model, erd, ...)
+- [ ] Update any documentation (javadoc, domain outcobra.server.model, erd, ...)
```

Comment:
Package name seems not really necessary

Reply:
Oh sorry I used the global replace function. I'll fix it